index.js

```diff
@@ -110,6 +111,13 @@ class GuideBot extends Client {
     return false;
   }
 
+  async reloadEvent(client, eventPath, eventName) {
+    delete require.cache[require.resolve(`${eventPath}${path.sep}${eventName}.js`)];
+    const event = new(require(`${eventPath}${path.sep}${eventName}.js`))(client);
+    client.on(eventName, (...args) => event.run(...args));
+    delete require.cache[require.resolve(`${eventPath}${path.sep}${eventName}.js`)];
```

Comment:
Removing the require after setting it?

Reply:
Similar to the comment above, I've yet to see this cause any sort of issues, and it's similar to how the current event loader functions as I said above. Can still remove the line if wanted.